pkg/netutil/netutil.go

```diff
@@ -92,3 +94,13 @@ func GetLocalIPs() ([]string, error) {
 
 	return localIPs, nil
 }
+
+// BaseURLFromFullURL returns the base url (http://hostname.example.com) from a URL, without port
+func BaseURLFromFullURL(fullURL string) string {
```

Comment:
Side note: this will be very useful to use when we have `ddev launch` written in go.